packages/grid_2d.py

```diff
@@ -0,0 +1,37 @@
+"""
+
+"""
+from __future__ import division, print_function
+import sys, os, pdb
+import numpy as np
+import nibabel as nib
+
+# def plot_mosaic(img_data, transpose=False):
+def plot_mosaic(img_data):
+    """ Return a mosaic plot for each slice of
```

Comment:
Check  https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt for a standard for documenting parameters and return values.   It's worth sticking to this standard because:

* other people are used to it, so they find it easy to read;
* there are standard processing tools that understand this format, and can make nice web pages and other documents using the format, to document your code.